src/interfaces/business-interfaces.ts

```diff
@@ -46,6 +56,7 @@ export interface ApiBusinessIF {
   lastLedgerTimestamp?: ApiDateTimeUtc // not used
   lastModified?: ApiDateTimeUtc // not used
   legalName: string
+  operatingName: string
```

Comment:
This property is not part of the business object, as far as I know. Did you find an example of where it is? (Then I'd fix my code.) Otherwise please remove it.